File: app/src/main/java/com/resonatestudios/pushupplus/controller/OpenHelper.java
package com.resonatestudios.pushupplus.controller;

import android.content.Context;
import android.database.sqlite.SQLiteDatabase;
import android.database.sqlite.SQLiteOpenHelper;

import androidx.annotation.Nullable;

public class OpenHelper extends SQLiteOpenHelper {
    private static final int DATABASE_VERSION = 1;
    private static final String DATABASE_NAME = "dbPushup.db";
    private static final String TABLE_CREATE =
            "CREATE TABLE HISTORY (" +
                    "ID INTEGER PRIMARY KEY AUTOINCREMENT," +
                    "AMOUNT INTEGER," +
                    "DURATION INTEGER," +
                    "DATE TEXT" +
                    ");";

    public OpenHelper(@Nullable Context context) {
        super(context, DATABASE_NAME, null, DATABASE_VERSION);
    }

    @Override
    public void onCreate(SQLiteDatabase db) {
        db.execSQL(TABLE_CREATE);
    }

    @Override
    public void onUpgrade(SQLiteDatabase db, int oldVersion, int newVersion) {
        db.execSQL("DROP TABLE IF EXISTS HISTORY");
    }
}

File: app/src/main/java/com/resonatestudios/pushupplus/controller/OpenHelper.kt
package com.resonatestudios.pushupplus.controller

import android.content.Context
import android.database.sqlite.SQLiteDatabase
import android.database.sqlite.SQLiteOpenHelper

class OpenHelper(context: Context?) : SQLiteOpenHelper(context, DATABASE_NAME, null, DATABASE_VERSION) {
    override fun onCreate(db: SQLiteDatabase) {
        db.execSQL(TABLE_CREATE)
    }

    override fun onUpgrade(db: SQLiteDatabase, oldVersion: Int, newVersion: Int) {
        db.execSQL("DROP TABLE IF EXISTS HISTORY")
    }

    companion object {
        private const val DATABASE_VERSION = 1
        private const val DATABASE_NAME = "dbPushup.db"
        private const val TABLE_CREATE = "CREATE TABLE HISTORY (" +
                "ID INTEGER PRIMARY KEY AUTOINCREMENT," +
                "AMOUNT INTEGER," +
                "DURATION INTEGER," +
                "DATE TEXT" +
                ");"
    }
}